101-vm-user-image-data-disks/azuredeploy.json

```diff
@@ -77,20 +77,32 @@
     "publicIPAddressName": "[concat(parameters('vmName'),'PublicIP')]",
     "vmName": "[parameters('vmName')]",
     "virtualNetworkName": "[concat(parameters('vmName'),'VNet')]",
-    "nicName": "[concat(parameters('vmName'),'Nic')]",
+    "nicName": "[concat(parameters('vmName'),'-nic1')]",
     "addressPrefix": "10.0.0.0/16",
     "subnet1Name": "Subnet-1",
     "subnet1Prefix": "10.0.0.0/24",
     "publicIPAddressType": "Dynamic",
     "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
     "subnet1Ref": "[concat(variables('vnetID'),'/subnets/',variables('subnet1Name'))]",
     "osDiskUrl": "[parameters('osDiskUrlParam')]",
-    "dataDiskUrl": "[parameters('dataDiskUrlParam')]",    
+    "dataDiskUrl": "[parameters('dataDiskUrlParam')]",
     "osDiskVhdName": "[concat(variables('osDiskUrl'),'-',variables('vmName'),'.vhd')]",
-    "dataDiskVhdName": "[concat(variables('dataDiskUrl'),'-',variables('vmName'),'.vhd')]"
+    "dataDiskVhdName": "[concat(variables('dataDiskUrl'),'-',variables('vmName'),'.vhd')]",
```

Comment:
This is a really odd convention for a filename - is this really the intent and a practice to recommend?